### PR TITLE
Allow base button to sit inline with text

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Allow base button to sit inline with text

## QA

- Run `gulp jekyll`
- Edit `examples/button.html` to add a chunk of inline text before the button element, like so;
```html
<span>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span><button>Button</button>
```
- View in browser: `http://127.0.0.1:4000/vanilla-framework/examples/base/button/` 
    

## Details

Fixes: #737

### CODE REVIEW

File: scss/_base_button.scss [hunk 1]
```diff
@@ -5,6 +5,10 @@
   button {
     @include button-pattern;
     line-height: 1rem;
+
+    span + & {
+      margin-left: .5rem;
```
Comment (anthonydillon) [inline scss/_base_button.scss]: We may want an entire `1rem` here. Difficult without a spec.

**Comment by anthonydillon** [comment]
Interesting idea to use a span to indicate "inlineness".

File: scss/_base_button.scss (hunk 1 above)
Comment (anthonydillon) [inline scss/_base_button.scss]: @yaili do you think it needs 16px or 8px on large screens?

Comment (barrymcgee) [inline scss/_base_button.scss]: With `.5rem` of margin-left: https://cl.ly/1k1W0J330q2r

With `1rem` of margin-left: https://cl.ly/2a2S46252z3t

Comment (yaili) [inline scss/_base_button.scss]: I think 1rem looks better. Should I create a design spec?

**Comment by barrymcgee** [comment]
@yaili Yes, if we think we need a spec, we probably do..

**Pull request closed without merging**